src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/CommandBarTests/UnoSamples_Tests.NativeCommandBar.cs

```diff
@@ -203,5 +203,25 @@ public void When_Navigated_CommandBarShouldKeepColor_Native_Frame()
 
 			ImageAssert.AreEqual(initial, final);
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.iOS)]
+		public void When_Navigated_CommandBarShouldHideBackButtonTitle_NativeFrame()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.CommandBar.BackButtonTitle.CommandBar_Frame");
+
+			_app.WaitForElement("NavigateInitialButton");
+			_app.FastTap("NavigateInitialButton");
+
+			_app.WaitForElement("NavigateToPage2Button");
+			_app.FastTap("NavigateToPage2Button");
+
+			_app.WaitForElement("Page2CommandBar");
+
+			var backButtonText = _app.Marked("Back").FirstResult();
```

Comment:
I don't recall, does .Marked("SomeText") return a TextBlock if its Text="SomeText"? In other words, would this really fail without the change?